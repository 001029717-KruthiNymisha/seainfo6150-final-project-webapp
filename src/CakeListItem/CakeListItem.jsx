import React from "react";
import CakeImage from "../CakeImage/CakeImage.jsx";
import styles from "./CakeListItem.module.css";

const CakeListItem = (props) => {
  return (
    <div className={styles.cakeListItem}>
      <CakeImage url = {props.url} title={props.title}></CakeImage>
      <div className={styles.titleStyle}>{props.title}</div>
    </div>
  );
};

export default CakeListItem;
/*<h1>{props.title}</h1>*/