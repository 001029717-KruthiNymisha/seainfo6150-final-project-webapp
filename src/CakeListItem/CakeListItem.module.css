.cakeListItem {
  display: flex;
  flex-direction: column;
  align-items: center;
  flex-wrap: wrap;
  justify-content: space-between;
  height: 100%;
}
.titleStyle{
font-family:cursive;
padding: 3px;
border: 1px solid;
background-color: #FFDAD6;
border-radius:5px;
}