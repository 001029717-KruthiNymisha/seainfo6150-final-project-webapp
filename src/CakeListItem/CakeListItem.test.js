import React from "react";
import { render } from "@testing-library/react";
import CakeListItem from "./CakeListItem.jsx";

it("renders the CakeListItem component correctly", () => {
  const { container } = render(
    <CakeListItem
      url="https://assets.wsimgs.com/wsimgs/rk/images/dp/wcm/202038/0804/img60o.jpg"
      title="Gluten-Free Happy Birthday Cake"
    />
  );
  expect(container).toMatchSnapshot();
});
