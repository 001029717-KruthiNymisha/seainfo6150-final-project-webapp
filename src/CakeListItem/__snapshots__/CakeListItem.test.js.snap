// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the CakeListItem component correctly 1`] = `
<div>
  <div
    class="cakeListItem"
  >
    <div
      class="cakeImage"
    >
      <img
        alt="Gluten-Free Happy Birthday Cake"
        src="https://assets.wsimgs.com/wsimgs/rk/images/dp/wcm/202038/0804/img60o.jpg"
      />
    </div>
    <div
      class="titleStyle"
    >
      Gluten-Free Happy Birthday Cake
    </div>
  </div>
</div>
`;
